dev/README_RELEASE_AIRFLOW.md

```diff
@@ -745,3 +746,15 @@ EOF
 ## Update Announcements page
 
 Update "Announcements" page at the [Official Airflow website](https://airflow.apache.org/announcements/)
+
```

Comment:
Should it be done "BEFORE" the release maybe?

Reply:
We should keep the config.yml file in good shape both in the main branch and in the releasing branch. This script only allows the validation of files on the master branch. The `added_version` field is set in releases to the correct values in branches while we merge cherry-pick commits.

Reply:
I mean - right now the process is to update confiig.yaml - after the release is published (and documentation generated) - should not we simply run the script before we update version and prepare changelog (this is already happening in the release branch). This was my proposal - to move it at the very beginning of the file.

Reply:
I think we have two problems: 1. updating the file in the main branch to include information about the new release, and 2. making sure that the new options in the release branch contain the value in the `added_version` field.  I am taking care of the first problem for now.

Reply:
Ah. I see. It's fetching from PYPI. fine for me then,